frameworks/kafka/tests/test_jmx.py

```diff
@@ -0,0 +1,163 @@
+
+        if self_signed_trust_store:
+            trust_store = '$MESOS_SANDBOX/jmx/trust_store'
+            trust_store_password = 'deleteme'
+        else:
+            trust_store = '$JAVA_HOME/lib/security/cacerts'
+            trust_store_password = 'changeit'
+
+        cmd = "export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre) && " \
+              "$JAVA_HOME/bin/java " \
+              "-Duser.home=$MESOS_SANDBOX " \
+              "-Djdk.tls.client.protocols=TLSv1.2 -Djavax.net.ssl.trustStore={trust_store} " \
+              "-Djavax.net.ssl.trustStorePassword={trust_store_password} " \
+              "-Djavax.net.ssl.keyStore=$MESOS_SANDBOX/jmx/key_store -Djavax.net.ssl.keyStorePassword=deleteme " \
+              "-Djavax.net.ssl.trustStoreType=JKS -Djavax.net.ssl.keyStoreType=JKS -jar jmxterm-1.0.1-uber.jar " \
+              "-l service:jmx:rmi:///jndi/rmi://$MESOS_CONTAINER_IP:31199/jmxrmi -u admin -p adminpassword " \
+              "-s -v silent -n".format(trust_store=trust_store, trust_store_password=trust_store_password)
+
+        input_jmx_commands = " < jmx_beans_command.txt"
+
+        full_cmd = "bash -c '{}{}'".format(cmd, input_jmx_commands)
+
+        _, output, _ = sdk_cmd.run_cli("task exec {} {}".format(broker_task_id_0, full_cmd), print_output=True)
+
+        assert "kafka.server:type=kafka-metrics-count" in output
+        assert "kafka.server:name=BrokerState,type=KafkaServer" in output
+
+        input_jmx_commands = " < jmx_domains_command.txt"
+        full_cmd = "bash -c '{}{}'".format(cmd, input_jmx_commands)
+        rc, output, stderr = sdk_cmd.run_cli("task exec {} {}".format(broker_task_id_0, full_cmd), print_output=True)
+
+        assert "kafka.server" in output
+        assert "kafka.controller" in output
+
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+        uninstall_jmx_secrets()
+
+
+def random_string(length=10):
+    letters = string.ascii_lowercase
+    return ''.join(random.choice(letters) for i in range(length))
+
+
+def write_to_file(content, file_path):
+    text_file = open(file_path, "w+")
+    text_file.write(content)
+    text_file.close()
+
+
+def generate_jmx_command_files(task_id: string):
+    cmd = "\n".join(
+        [
+            "echo beans >> jmx_beans_command.txt && ",
+            "echo domains >> jmx_domains_command.txt",
+        ]
+    )
+    full_cmd = "bash -c '{}'".format(cmd)
+    rc, _, _ = sdk_cmd.run_cli("task exec {} {}".format(task_id, full_cmd), print_output=True)
+    assert rc == 0, "Error creating jmx_commands file"
+
+
+def install_jmxterm(task_id: string):
+    jmx_term_url = 'https://downloads.mesosphere.io/jmx/assets/jmxterm-1.0.1-uber.jar'
+    cmd = "wget {}".format(jmx_term_url)
+    full_cmd = "bash -c '{}'".format(cmd)
+    rc, _, _ = sdk_cmd.run_cli("task exec {} {}".format(task_id, full_cmd), print_output=True)
+    assert rc == 0, "Error downloading jmxterm {}".format(jmx_term_url)
```

Comment:
Ditto.

Reply:
For example here just knowing that `wget` failed without knowing why would be a PITA.

Reply:
```
sdk_cmd.py                 235 INFO     STDERR:
Warning: The attached cluster is running DC/OS 1.13 but this CLI only supports DC/OS 1.12.
bash: whatever: command not found
```
you can check the logs for that so I don't see any issue here